common/djangoapps/coursetalk/tests/test_helpers.py

```diff
@@ -0,0 +1,16 @@
+""" CourseTalk widget helpers tests """
+from __future__ import unicode_literals
+from django.test import TestCase
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from coursetalk.helpers import get_coursetalk_course_key
+
+
+class GetCoursetalkCourseKeyTests(TestCase):
+    """
+    GetCoursetalkCourseKeyTests: test function get_coursetalk_course_key
+    """
+
+    def test_simple_key(self):
```

Comment:
There's some conditionals in `get_coursetalk_course_key` -- it would be good to test all code paths.